internal/tofu/context_plan2_test.go

```diff
@@ -7327,3 +7327,89 @@ func TestContext2Plan_importResourceWithSensitiveDataSource(t *testing.T) {
 		}
 	})
 }
+
+func TestContext2Plan_insuffient_block(t *testing.T) {
+	type testcase struct {
+		filename string
+		start    hcl.Pos
+		end      hcl.Pos
+	}
+
+	tests := map[string]testcase{
+		"insufficient-features-blocks-aliased-provider": {
+			filename: "provider[\"registry.opentofu.org/hashicorp/test\"] with no configuration",
+			start:    hcl.InitialPos,
+			end:      hcl.InitialPos,
+		},
+		"insufficient-features-blocks-nested_module": {
```

Comment:
Hmm, due to provider aliasing/defaulting, we don't know what module this occurred in.

Reply:
Given how provider resolution happens, I think this is as good as we can reasonably get.